Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbNativeWindow.cpp

```diff
@@ -219,17 +229,93 @@ namespace AzFramework
         }
     }
 
+    void XcbNativeWindow::DrawSplash()
+    {
+
+        int width = 0;
+        int height = 0;
+
+        auto get_image_data_static = [](int &width, int &height) -> uint8_t*
```

Comment:
I would stick to one naming convention, which seems to be camelCase in most cases in this code base.

Reply:
This is also a lambda used only once, below. Maybe its OK to used this code directly?

Reply:
This lambda pattern might be useful if someone will want to create some other strategy of loading image, and I don't think it creates a sensible division of code, for part that loads the image and part that displays it

Reply:
I don't think thats a practical reason to use a lambda for this purpose. If another strategy is applied, is it expected to create another lambda here?

Reply:
Please keep it simple

Reply:
Lambda removed